src/Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs

```diff
@@ -66,9 +67,10 @@ public IActionResult SignIn([FromRoute] string scheme)
             string loginHint,
             string domainHint,
             string claims,
-            string policy)
+            string policy,
+            [FromRoute] string scheme)
         {
-            string scheme = OpenIdConnectDefaults.AuthenticationScheme;
+            // scheme ??= OpenIdConnectDefaults.AuthenticationScheme;
```

Comment:
Good catch. Indeed, I think we'd want to remove this line

Reply:
i think it's okay, if there is no scheme, it will use the default. we can discuss though.